From 19b2c6e09a12f48a5caa14dd9a33fdabfc906741 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Thu, 8 Feb 2024 18:28:11 +0000
Subject: IWYU: usage of std::optional requires include in
 spare_render_process_host_manager.h

Bug: 957519
Change-Id: Ib492925af1f57ec10e42b567fead6c6a058efa00
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5259417
Commit-Queue: Charlie Reis <creis@chromium.org>
Reviewed-by: Charlie Reis <creis@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1258033}
---
 .../browser/renderer_host/spare_render_process_host_manager.h   | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/content/browser/renderer_host/spare_render_process_host_manager.h b/content/browser/renderer_host/spare_render_process_host_manager.h
index 9d2ba05e6e1a1..b7ebfd3bf9816 100644
--- a/content/browser/renderer_host/spare_render_process_host_manager.h
+++ b/content/browser/renderer_host/spare_render_process_host_manager.h
@@ -5,6 +5,8 @@
 #ifndef CONTENT_BROWSER_RENDERER_HOST_SPARE_RENDER_PROCESS_HOST_MANAGER_H_
 #define CONTENT_BROWSER_RENDERER_HOST_SPARE_RENDER_PROCESS_HOST_MANAGER_H_
 
+#include <optional>
+
 #include "base/callback_list.h"
 #include "base/functional/callback.h"
 #include "base/memory/raw_ptr.h"
-- 
2.40.1

